ci/common/common.sh

```diff
@@ -153,7 +153,7 @@ commit_subtree() {(
 
 with_token() {(
   set +x
-  set +o pipefail
+  set +e
```

Comment:
I think it might be better to step back and handle this differently.

From what I understood only a few commands would be wrapped with this to not leak the token?
What about using SSH (deploy) keys instead, for example?